components/layout_2020/display_list/mod.rs

```diff
@@ -65,9 +66,20 @@ impl<'a> DisplayListBuilder<'a> {
         }
     }
 
-    fn common_properties(&self, clip_rect: units::LayoutRect) -> wr::CommonItemProperties {
-        // TODO(gw): Make use of the WR backface visibility functionality.
-        wr::CommonItemProperties::new(clip_rect, self.current_space_and_clip)
+    fn common_properties(
+        &self,
+        clip_rect: units::LayoutRect,
+        style: &ComputedValues,
+    ) -> wr::CommonItemProperties {
+        // TODO(mrobinson): We should pass the hit test information here and not create
+        // so many hit test display items.
```

Comment:
The problem is that we want hit testing to be always present for each fragment, and always based on the border edge of the fragment. Some fragments do not have a border or background and thus do not generate a display item other than the hit-testing one. Some have backgrounds, but with the `background-clip` property such that the rectangle is not the one appropriate for hit testing.

We can still reduce the number of display items dedicated to hit testing when there happens to be another display item with the appropriate rectangle, but it’s not as straightforward as this comment suggests. Please reword it to indicate that.

Reply:
Sure. I'll update the comment to be a bit clearer.